pallets/ah-migrator/src/lib.rs

```diff
@@ -172,6 +193,10 @@ pub mod pallet {
 	pub type RcAccounts<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, RcAccountFor<T>, OptionQuery>;
 
+	/// The Asset Hub migration state.
+	#[pallet::storage]
+	pub type AhMigrationStage<T: Config> = StorageValue<_, MigrationStage, ValueQuery>;
```

Comment:
Not sure if `ValueQuery` makes sense if our default is `Pending` which says that the migration will start in the next block.

Reply:
I changed the doc for `Pending` in both pallets (...will start in the future). In both pallets it's `ValueQuery`. I think its better than the alternative with `OptionQuery` and no `Pending` variants at all (otherwise its never reachable stage).